generators/client-2/templates/_webpack.config.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+    entry: './src/main/webapp/app/app.main',
+    resolve: {
+        extensions: ['', '.ts', '.js', '.json'],
+        modules: ['./', 'node_modules']
+    },
+    output: {
+        path: './target/www/bin',
+        filename: 'app.bundle.js'
+    },
+    module: {
+        loaders: [{
+            test: /\.ts$/,
+            //exclude: /node_modules/,
+            loader: 'babel-loader!ts-loader'/*,
+            query: {
+                presets: ['es2015', 'angular2']
+            }*/
+        }],
+        rules: [
+            {
+                test: /\.ts$/,
+                exclude: [/\.(spec|e2e)\.ts$/],
+                loaders: [
+                    /*'awesome-typescript-loader',
```

Comment:
We should use the typescript loader I think, in order to compile our ts files. See https://github.com/angular/angular2-seed/blob/master/webpack.config.js